client/scss/styleguide/_styleguide.scss

```diff
@@ -25,30 +22,72 @@
   }
 }
 
-.styleguide__hex {
+.styleguide__palette__heading {
+  @include font('HNM4');
+  text-transform: uppercase;
+  flex-basis: 100%;
+  margin-bottom: 1em;
+}
+
+.styleguide__palette__section {
+  display: flex;
+  flex-wrap: wrap;
+  border-bottom: 3px solid color('cream');
+  padding-bottom: 15px;
+  margin-bottom: 30px;
+
+  &:last-child {
+    border: 0;
+  }
+}
+
+.styleguide__palette__block {
+  flex-basis: 25%;
+  margin-bottom: 15px;
+}
+
+.styleguide__palette__color {
   position: relative;
-  width: 200px;
-  height: 200px;
+  min-width: 200px;
+  margin-right: 15px;
+  border: 1px solid color('transparent');
+
+  &--white {
+    border-color: color('accessible-grey-2');
+  }
+
+  &:before {
+    content: '';
+    display: block;
+    padding-top: 100%;
+  }
 }
 
-.styleguide__color {
-  position: absolute;
-  top: 5px;
-  left: 5px;
-  background: color('white');
-  padding: 5px;
-  max-width: 190px;
-  border-radius: 3px;
-  border: 1px solid color('smoke');
+.styleguide__palette__hyphen {
+  visibility: hidden;
+}
+
+.styleguide__palette__name {
```

Comment:
The key benefit of BEM as I see it is to minimise reliance on the cascade, in order to minimise issues arising from classes with more specificity than necessary, so avoiding nesting where possible is a good thing.

You're right, it's slightly moot here, but I think there's still a benefit in having any styleguide-specific classes prefixed, so there's absolutely no ambiguity in our markup regarding what is and isn't required in the app and styleguide.